Dockerfile

```diff
@@ -9,6 +9,8 @@ RUN \
     apt-get update -y && \
       DEBIAN_FRONTEND=noninteractive apt-get install -y \
       autoconf \
+      gcc-5 \
+      g++-5 \
```

Comment:
These changes line 12 and 13 are not required. The build is compatible with gcc 7 also

Reply:
thanks. removed at https://github.com/peerplays-network/peerplays/pull/212/commits/a07bcad7e80877b2a63fff73d1175c9f7ae477af